### PR TITLE
Postgres with pulsar

### PR DESCRIPTION
**Pull request by skyrocknroll**
Added pulsar example for postgres

### CODE REVIEW

File: apache-pulsar/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Streaming Postgres Database Changes to Apache pulsar Using Debezium
```
Comment (gunnarmorling) [inline apache-pulsar/README.md]: Could you expand the README a bit, so it also discusses how to start a Pulsar broker (e.g. via Docker) and how to consume messages sent from Debezium, e.g. using something similar to Kafka Connect's console consumer?

File: apache-pulsar/src/main/java/io/debezium/examples/apache/pulsar/config/PropertyLoader.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import java.util.Enumeration;
+import java.util.Properties;
+
+
+
+public class PropertyLoader {
+
+
+    public static void loadEnvironmentValues(Properties properties) {
+        System.out.println("=================================");
```
Comment (gunnarmorling) [inline apache-pulsar/src/main/java/io/debezium/examples/apache/pulsar/config/PropertyLoader.java]: It'd be nic to get rid of the sys-out calls here and in some other places.

**Comment by skyrocknroll** [comment]
Hi, @gunnarmorling I have implemented the review comments. Please review it.

**Comment by gunnarmorling** [comment]
Superseded by #29 which I just merged after doing a bit of clean-up here and there. Thanks a lot for this contribution, @skyrocknroll! Very nice to have a demo for Pulsar!

**Pull request closed without merging**